src/components/Main/Footer/Footer.tsx

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 import Contact from './Contact';
 import Links from './Links';
+import footerWaves from '../../../assets/images/footer-wave.svg';
+import logo from '../../../assets/images/peacock-logo.svg';
+import {Row, Col} from 'react-bootstrap';
 
-// TODO: Add top graphic (should be same as header, but flipped)
-// TODO: Add logo
-// TODO: Add contact
-// TODO: Add links (Brand Manual should redirect to Figma since we haven't coded it)
 interface Props {
   id: string;
 }
 
+
 function Footer(props: Props) {
   return (
     <div id={props.id}>
-      <img src="#" alt="Girls Who Code at the University of Virginia logo" />
-      <Contact />
-      <Links />
+      <img src={footerWaves} style={{ transform: 'rotate(180deg)', marginBottom:-1}} alt="Orange wave"/>
+      <div className="bg-orange d-flex flex-row" style={{paddingBottom: 50, paddingTop: 60}}>
+        <Col md={4} sm={3}>
+          <img src={logo} width="50%" style={{paddingLeft:"10%"}} alt="Girls Who Code at the University of Virginia logo"/>
```

Comment:
please make this fixed width, reference `Navbar.tsx` for an example - exact widths are on the figma